src/Uno.UI/UI/Xaml/Application.macOS.cs

```diff
@@ -76,7 +78,19 @@ private ApplicationTheme GetDefaultSystemTheme()
 			else
 			{
 				return ApplicationTheme.Dark;
-			}			
+			}
+		}
+
+		private void SetCurrentLanguage()
+		{
+			try
+			{
+				var language = NSLocale.PreferredLanguages[0];
+				var cultureInfo = CultureInfo.CreateSpecificCulture(language);
+				CultureInfo.CurrentUICulture = cultureInfo;
+				CultureInfo.CurrentCulture = cultureInfo;
+			}
+			catch (Exception) { }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Handle the exception or explain in a comment why it can be ignored.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5521872)

Reply:
We may want to log an error there as well.